pom.xml

```diff
@@ -3,19 +3,44 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>org.codice.ddf</groupId>
-    <artifactId>jsonrpc-api</artifactId>
-    <version>1.0.0</version>
+    <groupId>org.connexta</groupId>
+    <artifactId>revelio</artifactId>
+    <version>0.1-SNAPSHOT</version>
     <packaging>bundle</packaging>
+    <name>Connexta :: Revelio</name>
 
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <ddf.version>2.18.0</ddf.version>
         <opengis.version>19.1</opengis.version>
+        <releases.repository.url />
+        <snapshots.repository.url />
     </properties>
 
+    <scm>
+        <url>https://github.com/connexta/revelio</url>
+        <connection>scm:git:https://github.com/connexta/revelio.git</connection>
+        <developerConnection>scm:git:https://github.com/connexta/revelio.git</developerConnection>
+        <tag>HEAD</tag>
```

Comment:
Can we point to a fixed version. Determinism would be really nice. Then we can know what version of GSR exposes which version of the jsonrpc-api.

Reply:
Ignore this :laughing: